src/Banner.js

```diff
@@ -0,0 +1,37 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import styled from '@emotion/styled'
+import F from 'futil'
+import { defaultProps } from 'recompose'
+import { Icon, Text, Flex } from '.'
+import theme from './theme'
+
+let getColor = F.aliasIn(theme.colors)
+
+let bannerStyle = (bgColor, textColor) => ({
+  backgroundColor: getColor(bgColor),
+  '*': { color: `${getColor(textColor)} !important` },
```

Comment:
In CSS, `*` is generally a bad practice for performance reasons. Does that apply here too?

Reply:
it does since it generates a css `*` selector, but hopefully to a lesser extent because emotion applies individual classNames to each component.

you're right though, we should probably avoid it unless absolutely necessary